docs/interfaces/c61/modem.md

```diff
@@ -0,0 +1,93 @@
+---
+title: C61 Modem
+tags:
+    - C61
+    - CT GL
```

Comment:
I think it's strange to list CT GL when the title is C61

Reply:
Alternative is to copy all to CT GL folder?

Reply:
OK, if want :-)